src/styles/components/_product-card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $color-grey-light;
+  background: $color-white;
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
i think will be good set only `font-family` only one time in `body`

Reply:
Assigned class .page for body and added fonts for it